src/services/ui/src/pages/actions/setups/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./_"; // Types
```

Comment:
A new index for form setups. A setup is simply the `schema` and `attachments` pairing as these are the dynamic pieces necessary for altering the form's requirements per plan type.